### ISSUE TITLE
Add swarm support

### ISSUE THREAD

**Issue by niriak** [fabric8io/docker-maven-plugin#407]
Trying to start a docker container on a docker swarm server results in an error message:

````
[output of mvn with -X]
----------------------------- 8< ---------------------------
[ERROR] Failed to execute goal io.fabric8:docker-maven-plugin:0.14.3-SNAPSHOT:start (start blackbox container) on project test: Execution start blackbox container of goal io.fabric8:docker-maven-plugin:0.14.3-SNAPSHOT:start failed: JSONObject["Warnings"] not found. -> [Help 1]
org.apache.maven.lifecycle.LifecycleExecutionException: Failed to execute goal io.fabric8:docker-maven-plugin:0.14.3-SNAPSHOT:start (start blackbox container) on project test: Execution start blackbox container of goal io.fabric8:docker-maven-plugin:0.14.3-SNAPSHOT:start failed: JSONObject["Warnings"] not found.
        at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:212)
        at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:153)
        at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:145)
        at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject(LifecycleModuleBuilder.java:116)
        at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject(LifecycleModuleBuilder.java:80)
        at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build(SingleThreadedBuilder.java:51)
        at org.apache.maven.lifecycle.internal.LifecycleStarter.execute(LifecycleStarter.java:128)
        at org.apache.maven.DefaultMaven.doExecute(DefaultMaven.java:307)
        at org.apache.maven.DefaultMaven.doExecute(DefaultMaven.java:193)
        at org.apache.maven.DefaultMaven.execute(DefaultMaven.java:106)
        at org.apache.maven.cli.MavenCli.execute(MavenCli.java:863)
        at org.apache.maven.cli.MavenCli.doMain(MavenCli.java:288)
        at org.apache.maven.cli.MavenCli.main(MavenCli.java:199)
        at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.lang.reflect.Method.invoke(Method.java:606)
        at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced(Launcher.java:289)
        at org.codehaus.plexus.classworlds.launcher.Launcher.launch(Launcher.java:229)
        at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode(Launcher.java:415)
        at org.codehaus.plexus.classworlds.launcher.Launcher.main(Launcher.java:356)
Caused by: org.apache.maven.plugin.PluginExecutionException: Execution start blackbox container of goal io.fabric8:docker-maven-plugin:0.14.3-SNAPSHOT:start failed: JSONObject["Warnings"] not found.
        at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo(DefaultBuildPluginManager.java:145)
        at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:207)
        ... 20 more
Caused by: org.json.JSONException: JSONObject["Warnings"] not found.
        at org.json.JSONObject.get(JSONObject.java:473)
        at io.fabric8.maven.docker.access.hc.DockerAccessWithHcClient.logWarnings(DockerAccessWithHcClient.java:402)
        at io.fabric8.maven.docker.access.hc.DockerAccessWithHcClient.createContainer(DockerAccessWithHcClient.java:164)
        at io.fabric8.maven.docker.service.RunService.createAndStartContainer(RunService.java:98)
        at io.fabric8.maven.docker.StartMojo.executeInternal(StartMojo.java:85)
        at io.fabric8.maven.docker.AbstractDockerMojo.execute(AbstractDockerMojo.java:164)
        at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo(DefaultBuildPluginManager.java:134)
        ... 21 more
````

**Comment by niriak** [comment]
Examples of "docker create" result body:

- docker server 1.10:
{"Id":"5293023130c34b2efa60d755bd13b377ae8ab92464140980f9f8fbfe7fe947fc","Warnings":null}

- docker swarm server 1.13:
{"Id":"a23a523268ccad5c761e9ccd1cf21467d894de0ecc65002adce6509250ef804f"}

**Comment by rhuss** [comment]
Thanks for spotting this. Its fixed now and I pushed out a 0.14.3-SNAPSHOT if you want to try it out ...

**Comment by niriak** [comment]
It works.
Thanks a lot!
